lib/client/identityfile/identity.go

```diff
@@ -344,6 +363,104 @@ func Write(cfg WriteConfig) (filesWritten []string, err error) {
 	return filesWritten, nil
 }
 
+func writeCassandraFormat(cfg WriteConfig, writer ConfigWriter) ([]string, error) {
+	if cfg.JKSPassword == "" {
```

Comment:
The JKSPassword is optional but so by default it will be generated by us but I can imagine a case when your cert expired and you want to generate a jks file with the same password to preserve the cassanda configuration file and replace only certs on disk.